### DISCUSSION TITLE
macOS 14 Sonoma

### DISCUSSION THREAD

**Discussion by SkyrilHD** [SkyrilHD/Dell-E7250-Hackintosh#70] [Show and tell]
Using the Ventura EFI, I was able to Sonoma without any issues. Since there is no GPU acceleration, it is quite unusable. Now, we need to wait until the dortania team adds support for Sonoma :)

![Sonoma on E7250](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/assets/28839925/ba123009-0f70-4979-a16e-47fadadab998)

**Comment by n2ng-dev** [comment]
This EFI Support Sonoma ?

**Reply by 1rocketdude** [reply]
sonoma 14.2.1 works for me

**Comment by SkyrilHD** [comment]
macOS Sonoma Release Candidate is working! However, I experience issues when updating with root patches. Clean install is needed. I will update the EFI shortly, but release will be on the same day as Sonoma gets released

![Bildschirmfoto 2023-09-13 um 16 08 52](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/assets/28839925/46671ceb-d306-4f1e-a499-98b371116b73)

**Reply by SkyrilHD** [reply]
The new EFI is now public on the [14.0 branch](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/tree/14.0) and can be tested!

**Reply by n2ng-dev** [reply]
How to download and create installer Sonoma Release Candidate ?

**Reply by SkyrilHD** [reply]
> How to download and create installer Sonoma Release Candidate ?

1. Download the InstallAssistant.pkg from Apple's servers (cannot link to a direct download, but you may find the links on osxdaily)
2. Install the .pkg then you should see the Install app in /Applications
3. Open Terminal and type the following command: `sudo /Applications/Install\ macOS\ Sonoma\ beta.app/Contents/Resources/createinstallmedia --volume /Volumes/MyVolume`
4. After completion, mount the EFI partition of the USB drive and copy the Sonoma EFI to the partition.
5. Adjust the config.plist for your device (SN, MLB, Intel WiFi, Touchscreen)
6. Boot the Sonoma installer
7. Proceed installation as usual
8. After installation, download the nightly version of OCLP from pull request #1077 found on the OCLP repo
9. Open OCLP and run the "Post-Install Root Patch"
10. After completion, reboot and enjoy Sonoma

**Comment by 1rocketdude** [comment]
this just worked. thank you. Extend life of a perfectly good laptop that is super easy to repair and upgrade.
I would add one note in the documentation to make explicit. Installing Sonoma (I would guess Ventura as well) does not overwrite older installation user directories. This saved me a huge amount of time as no need to restore from a backup. I "upgraded" from BigSur. I simply copied the Sonoma EFI folder to my EFI partition, booted the USB install for Sonoma, answered all the questions, came back an hour later, and magically had 14.2.1 running.

**Reply by 1rocketdude** [reply]
even Sidecar works! Brilliant.

**Comment by madilsharief** [comment]
This doesn't work. I have some old EFI for monetory it boots up the OS with isues but this one doesn't work. It does not show any internal harddrive on installer or in diskutils in install time. (i am using Sonoma 14.3 offline installer)

**Reply by SkyrilHD** [reply]
Press space bar in the OpenCore menu.

**Reply by madilsharief** [reply]
That would bring up reset NVM and other options but i am talknig about the time when i am successfully in recovery and i use diskUtils and see no internal hard drive. 

![WhatsApp Image 2024-05-01 at 13 11 50_64e40539](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/assets/68893283/7a4fee4b-4965-4fc9-ae43-206cab7ff68b)

**Reply by madilsharief** [reply]
Ok that was my bad. In BIOS HDD Mode was set to RAID thats why some old EFI was able to see it but this one did not. Just set it to AHCI and now it booted without any issue in one time. Thank You for sharing such gr8 EFI with gr8 effort.
